finder-app/abc.txt

```diff
@@ -0,0 +1 @@
+sdgsg
```

Comment:
@rajatchaple this is probably something you don't need to add to your repository

Reply:
Removed